Lib/site.py

```diff
@@ -194,7 +194,7 @@ def addpackage(sitedir, name, known_paths):
                 print("Error processing line {:d} of {}:\n".format(n+1, fullname),
                       file=sys.stderr)
                 import traceback
-                for record in traceback.format_exception(*sys.exc_info()):
+                for record in traceback.format_exc():
                     for line in record.splitlines():
```

Comment:
TIL that they are not actually equivalent:

```
(Pdb) pp traceback.format_exception(*sys.exc_info())
['Traceback (most recent call last):\n',
 '  File "/Users/iritkatriel/src/cpython/Lib/site.py", line 186, in '
 'addpackage\n'
 '    exec(line)\n',
 '  File "<string>", line 1, in <module>\n',
 "ModuleNotFoundError: No module named 'nosuchmodule'\n"]
(Pdb) pp traceback.format_exc()
('Traceback (most recent call last):\n'
 '  File "/Users/iritkatriel/src/cpython/Lib/site.py", line 186, in '
 'addpackage\n'
 '    exec(line)\n'
 '  File "<string>", line 1, in <module>\n'
 "ModuleNotFoundError: No module named 'nosuchmodule'\n")

```

Reply:
I don’t actually see the difference?

Reply:
string vs list of strings